lib/takeaway.rb

```diff
@@ -0,0 +1,100 @@
+require_relative 'order'
+
+class Takeaway
+
+  attr_accessor :active_order, :order_items, :menu_item, :kitchen, :cooking
+
+    def initialize
+      @kitchen = Kitchen.new
+        menu
+    end
+
```

Comment:
could you inject a new instance of kitchen rather than hard coding it in? i.e. def initialize(kitchen)
@kitchen = kitchen and you just pass in Kitchen.new as the argument?